dashboards/lodestar_networking.json

```diff
@@ -104,6 +104,7 @@
               "tooltip": false,
               "viz": false
             },
+            "insertNulls": false,
```

Comment:
Not sure if it is correct to drop these, it was added by grafana when migrating the dashboard from 9.3.2 to 10.1.1. The diff is kinda annoying but it will happen only once for each dashboard until we update grafana to newer version again.

I feel like this is a never ending battle of us dropping new properties and grafana adding them, maybe the best solution would be to update all dashboards once after a grafana upgrade. This can be done by saving each dashboard once and running download script.

Reply:
also works for me, but let's do a single PR populating for all dashboards